src/llmcompressor/transformers/finetune/text_generation.py

```diff
@@ -63,6 +53,18 @@ def oneshot(**kwargs) -> None:
     oneshot(**kwargs)
 
 
+@deprecated(
+    message=(
+        "`from llmcompressor import train` is deprecated, "
+        "please use `from llmcompressor import train`."
+    )
+)
+def train(**kwargs):
+    from llmcompressor import train
```

Comment:
apart from the stage runner functionality, why do we need the steps covered in main?

Reply:
Thats it, just stages!